ccd-definition/CaseEventToComplexTypes/draftSDO-parentsAndRespondentsCustom.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Minor - this filename is not consistent with others and is displayed before the "main" one.

Reply:
the file names match the CaseFieldID. CaseFieldID has a 30 character limit. parentsAndRespondentsCustomDirections runs over. What would you like me to do?

Reply:
Ah ok, makes sense then, was not aware of the 30 limit on the case field...